src/Perspex.Controls/GridSplitter.cs

```diff
@@ -2,42 +2,240 @@
 // Licensed under the MIT license. See licence.md file in the project root for full license information.
 
 using System;
+
+using Perspex.Collections;
 using Perspex.Controls.Primitives;
 using Perspex.Input;
-using Perspex.Rendering;
 using Perspex.VisualTree;
 
 namespace Perspex.Controls
 {
+    /// <summary>
+    /// Unlike WPF GridSplitter, Perspex GridSplitter has only one Behavior, GridResizeBehavior.PreviousAndNext
+    /// </summary>
     public class GridSplitter : Thumb
     {
-        private Grid _grid;
+        /// <summary>
+        /// Defines the <see cref="Orientation"/> property.
+        /// </summary>
+        public static readonly PerspexProperty<Orientation?> OrientationProperty =
+            PerspexProperty.Register<GridSplitter, Orientation?>(nameof(Orientation));
+
+        protected Grid _grid;
+
+        private IGridColumnsResizer _resizer;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="GridSplitter"/> class.
+        /// Gets or sets the orientation of the GridsSlitter, if null, it's inferred from column/row defenition(should be auto).
```

Comment:
Typo in definition. Also probably move the "if null..." etc stuff to the remarks section.